src/api/resources/openApi.json

```diff
@@ -795,6 +795,17 @@
           "example": [135.5, -25.2]
         }
       },
+      "ZoomParam": {
+        "in": "query",
+        "name": "zoom",
+        "description": "Limit response to entries that should only appear above the supplied zoom level.",
```

Comment:
Really nitpick, but the provider is free to apply whatever logic it wants with the Zoom value, not just limit entries.

Is it obvious from the context or should what the value is be documented? I mean that it is referring to the zoom level of the view = map.